packages/git/src/browser/git-view-contribution.ts

```diff
@@ -104,6 +104,30 @@ export namespace GIT_COMMANDS {
     export const DISCARD_ALL = {
         id: 'git.discard.all'
     };
+    export const STASH = {
+        id: 'git.stash',
+        label: 'Stash'
+    };
+    export const APPLY_STASH = {
+        id: 'git.stash.apply',
+        label: 'Apply Stash...'
+    };
+    export const APPLY_LATEST_STASH = {
+        id: 'git.stash.apply.latest',
+        label: 'Apply Latest Stash'
+    };
+    export const POP_STASH = {
+        id: 'git.stash.pop',
+        label: 'Pop Stash...'
+    };
+    export const POP_LATEST_STASH = {
+        id: 'git.stash.pop.latest',
+        label: 'Pop Latest Stash'
```

Comment:
Other Git commands follow the ` label: 'Git: Your Command'` pattern. Did you omit the prefix on purpose?

Reply:
No, there wasn't any special purpose why I omitted it. I can change the label and slice it again. But, can you explain why it is like that? I guess I missed the purpose of this pattern.

Reply:
Here is the commit I found: https://github.com/theia-ide/theia/commit/b5bf376f54964da2e1ca839f87d294039b74fede
And the corresponding GH issue: https://github.com/theia-ide/theia/issues/1119

Reply:
I assume it is for the _Command Pallett_ too see some context for your command.

Reply:
If I read [this comment](https://github.com/theia-ide/theia/issues/1119#issuecomment-372812915) it seems to me as if it is not important for the stash commands, since they are only in Git context.

Reply:
![screen shot 2019-02-26 at 16 53 07](https://user-images.githubusercontent.com/1405703/53426285-ffe57580-39e6-11e9-8fd1-971cf2fa9e44.jpg)

?

Reply:
*Sigh* ... you're right. I totally overlooked that possibility. I will change it...

Reply:
@jbicker you can use the `category` property. It is used only for prefixing in the the `quick open` and does not affect menus so you don't need to slice.

https://github.com/theia-ide/theia/blob/9fe8aa63bb2b2568abbd5f7e49b36b57c49e2a49/packages/core/src/common/command.ts#L42